packages/ui5-proxy-middleware/src/i18n.ts

```diff
@@ -0,0 +1,34 @@
+import i18next, { TOptions } from 'i18next';
+import translations from './translations/ui5-proxy-middleware.i18n.json';
+
+const NS = 'ui5-proxy-middleware';
+
+/**
+ * Initialize i18next with the translations for this module.
+ */
+export async function initI18n(): Promise<void> {
+    await i18next.init({
+        resources: {
+            en: {
+                [NS]: translations
+            }
+        },
+        lng: 'en',
+        fallbackLng: 'en',
+        defaultNS: NS,
+        ns: [NS]
+    });
+}
+
+/**
+ * Helper function facading the call call to i18next.
+ *
+ * @param key i18n key
+ * @param options additional options
+ * @returns {string} localized string stored for the given key
+ */
+export function t(key: string, options?: TOptions): string {
```

Comment:
Why do we have this wrapper function?

Reply:
Good question, I just copied it from one of the other packages 😅

Reply:
Ah, ok. If memory serves, that wrapper was used to always escape HTML entities and later was removed because this can be done on a case-by-case basis in the template itself (or during invocation). Probably outlived its usefulness.